### PR TITLE
Compatible with latest version of Atom

### PR DESCRIPTION
**Pull request by Announcement**
# Update
## Atom 1.2.4

#32 #30 #31 #29 #28 #27
Your issues are now all fixed. 

#20 #19
Your issues are because of iFrame. 
It is resolvable with basically proxying, buuuuttt... yeah. No thanks. Respond and if you really insist I'll implement an AJAX + Proxy solution to stop the security errors and let you browser Tumblr "Seamlessly" within atom.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 node_modules
+bin
```
Comment (Announcement) [inline .gitignore]: for the gruntfile.js output

File: README.md [hunk 2]
```diff
@@ -2,7 +2,43 @@
 
 Opens a web browser in an editor in Atom.
 
-* `ctrl-shift-G` to open a site
```
Comment (Announcement) [inline README.md]: Removed because of collisions, actually hits a core packaged.

File: lib/web-editor-uri-mini-view.coffee [hunk 3]
```diff
@@ -1,70 +1,11 @@
-validator = require 'validator'
 Point = require 'atom'
-{$, View, TextEditorView} = require 'atom-space-pen-views'
```
Comment (Announcement) [inline lib/web-editor-uri-mini-view.coffee]: Removed the deprecated protocol/application programming interface entirely. Nothing is require(d) except 'atom'

File: lib/web.coffee [hunk 4]
```diff
@@ -1,23 +1,93 @@
-_ = require 'underscore-plus'
-url = require 'url'
-_str = require 'underscore.string'
+# atom clean up work
+{CompositeDisposable} = require 'atom'
```
Comment (Announcement) [inline lib/web.coffee]: Removed the deprecated protocol/application programming interface entirely. Nothing is require(d) except 'atom'

File: package.json [hunk 5]
```diff
@@ -1,18 +1,25 @@
 {
   "name": "web-view",
-  "version": "0.3.0",
-  "description": "Opens a web browser in Atom",
+  "version": "1.2.4",
```
Comment (Announcement) [inline package.json]: Version of Atom :)

**Comment by chamini2** [comment]
Why hasn't this been merged? I would really like this package to be usable.

**Comment by gabceb** [comment]
Sorry for the delay. Ill review and merge this week.

**Comment by chamini2** [comment]
Apologies for being insistent, just reminding you to check this out when possible.

**Comment by Taconut** [comment]
Nothing yet?

**Comment by gabceb** [comment]
I've published a new major version based on these changes. Sorry for the delay and thanks!

**Pull request merged**